OptiFineDoc/assets/minecraft/optifine/lang/pl_pl.lang

```diff
@@ -319,6 +419,12 @@ of.options.CUSTOM_ENTITY_MODELS.tooltip.3=  Wył. - opcja wyłączona, szybciej
 of.options.CUSTOM_ENTITY_MODELS.tooltip.4=Własne modele są pobierane z
 of.options.CUSTOM_ENTITY_MODELS.tooltip.5=aktualnej paczki tekstur.
 
+of.options.CUSTOM_GUIS=Własne GUI
+of.options.CUSTOM_GUIS.tooltip.1=Własne GUI
+of.options.CUSTOM_GUIS.tooltip.2=  Wł. - własne GUI (domyślne), niższa wydajność
+of.options.CUSTOM_GUIS.tooltip.3=  Wył. - domyślne GUI, wyższa wydajność
+of.options.CUSTOM_GUIS.tooltip.4=Własne GUI są dostarczane z obecnej paczki zasobów.
```

Comment:
Correction:

of.options.CUSTOM_GUIS.tooltip.4=Własne GUI jest dostarczane z obecnej paczki zasobów.